README.md

```diff
@@ -3,33 +3,35 @@
   <br /><br />
 
   [![CI Status](https://github.com/elemaudio/elementary/actions/workflows/main.yml/badge.svg)](https://github.com/elemaudio/elementary/actions)
-  ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
   [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/elemaudio/elementary/blob/main/LICENSE.md)
   [![Discord Community](https://img.shields.io/discord/826071713426178078?label=Discord)](https://discord.gg/xSu9JjHwYc)
+  [![npm installs](https://img.shields.io/npm/dt/%40elemaudio/core?label=npm%20installs&color=%23f472b6)](https://www.npmjs.com/package/@elemaudio/core)
```

Comment:
This is my bad I think– would you mind dropping the changes to README here so that the PR only affects the compressor code? I'll fix up the README changes in a follow-up commit